eng/pipelines/coreclr/templates/xplat-pipeline-job.yml

```diff
@@ -103,6 +104,18 @@ jobs:
         - name: priorityArg
           value: '-priority=1'
 
+    - librariesArtifactName: ''
+    - librariesOverrideArg: ''
+    - librariesDownloadDir: ''
+
+    - ${{ if ne(parameters.liveLibrariesBuildConfig, '') }}:
+      - librariesArtifactName: ${{ format('libraries_bin_{0}_{1}{2}_{3}_{4}', 'netcoreapp', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveLibrariesBuildConfig) }}
```

Comment:
It might make sense to move these `artifactName` variables to xplat-setup template as it is used to upload and download and we already have all the required parameters at that point. Anyway, that can be done in a follow up PR, just calling it out so that we don't forget.